src/kirby/components/item/item.component.scss

```diff
@@ -58,6 +58,30 @@
   }
 }
 
+:host-context(kirby-list ion-item-sliding:first-of-type) {
```

Comment:
Pretty sure this won't work when compiled with `aot` because of this issue:
https://github.com/angular/angular-cli/issues/13854#issuecomment-470831308
See [button.component.scss](https://github.com/kirbydesign/designsystem/blob/b7e56696090a6cbf360a3b3fdd4a0138579a0fe0/src/kirby/components/button/button.component.scss#L190)